internal/suggester/suggester_test.go

```diff
@@ -54,6 +54,10 @@ func Test_GivenIncompleteKeywordInput_WhenSuggestCompletion_ExpectValidKeywords(
 		// 	input:              "insert i",
 		// 	expectedSuggestion: []string{"nto"},
 		// },
+		{
+			input:              " ",
+			expectedSuggestion: nil,
```

Comment:
This isn't a proper test because `""` wouldn't trigger suggestion either. We need something that we know that would be completed if wasn't by the space. Something like `"inse "`

Reply:
probably a test that check if suggestions has space as the first character also

Reply:
Fixed!